lib/curl_ntlm_core.h

```diff
@@ -53,6 +53,10 @@
 #define USE_NTLM2SESSION 1
 #endif
 
+#if !defined(USE_NTLM2SESSION)
+#define USE_NTLM2SESSION 0
```

Comment:
So what's the exact warning this fixes? I only see a `!defined(USE_NTLM2SESSION)` use of this and that looks perfectly fine to me?